central/graphql/resolvers/vulnerability_requests_test.go

```diff
@@ -228,9 +226,9 @@ func (s *VulnRequestResolverTestSuite) createComponentCVEDataStore(_ *dackbox.Da
 }
 
 func (s *VulnRequestResolverTestSuite) createVulnRequestDataStore(pendingReqCache, activeReqCache vulnReqCache.VulnReqCache) {
-	ds, err := vulnReqDS.NewForTestOnly(s.T(), s.db, s.bleveIndex, pendingReqCache, activeReqCache)
-	s.NoError(err)
-	s.vulnReqDataStore = ds
+	//ds, err := vulnReqDS.NewForTestOnly(s.T(), s.db, nil, pendingReqCache, activeReqCache)
+	//s.NoError(err)
+	//s.vulnReqDataStore = ds
```

Comment:
why commenting and not removing?

Reply:
I just commented to keep the test looking the same and not deleting. There are tickets for porting these tests to Postgres, but didn't want to just delete the code and then the test may be more difficult to port